Source/Core/Core/ConfigLoaders/NetPlayConfigLoader.cpp

```diff
@@ -100,10 +102,10 @@ class NetPlayConfigLayerLoader final : public Config::ConfigLayerLoader
         layer->Set(Config::MAIN_GCI_FOLDER_A_PATH_OVERRIDE, path + "Card A");
         layer->Set(Config::MAIN_GCI_FOLDER_B_PATH_OVERRIDE, path + "Card B");
 
-        const std::string file = File::GetUserPath(D_GCUSER_IDX) + GC_MEMCARD_NETPLAY + "%c." +
-                                 m_settings.m_SaveDataRegion + ".raw";
-        layer->Set(Config::MAIN_MEMCARD_A_PATH, StringFromFormat(file.c_str(), 'A'));
-        layer->Set(Config::MAIN_MEMCARD_B_PATH, StringFromFormat(file.c_str(), 'B'));
+        const std::string file = fmt::format("{}{}{}.{{}}.raw", File::GetUserPath(D_GCUSER_IDX),
```

Comment:
I think this should be

```suggestion
        const std::string file = fmt::format("{}{}{{}}.{}.raw", File::GetUserPath(D_GCUSER_IDX),
```

instead?

(IMO this is a bit hard to read... Any thoughts on having a small function/lambda that takes the memcard letter and returns the whole path?)

Reply:
Done